packages/firestore/src/core/view.ts

```diff
@@ -37,6 +37,7 @@ import {
   SyncState,
   ViewSnapshot
 } from './view_snapshot';
+import { OnlineState } from './types';
```

Comment:
Does prettier not sort imports (by location in `from`)?

Reply:
manually fixed.